src/share/doc/javatest/regtest/tag-spec.html

```diff
@@ -254,11 +254,14 @@ <h2 id="DECLARATIVE_TAGS">DECLARATIVE TAGS</h2>
 
 <p>In general, classes in library directories are not automatically compiled
 as part of a compilation command explicitly naming the source files containing
-those classes. A test that relies upon library classes should contain appropriate
-<code>@build</code> directives to ensure that the classes will be compiled.
-It is strongly recommended that tests do not rely on the use of implicit compilation
-by the Java compiler. Such an approach is generally fragile, and may lead to
-incomplete recompilation when a test or library code has been modified.
+those classes. The library directory is used as additional sourcepath for test compilation.
+So jtreg implicitly compiles all test dependencies that are located in library directory.
+A test that relies upon library classes might contain appropriate
+<code>@build</code> directives to explictily point classes to be compiled.
+This might be useful when test depends on library classes during runtime only.
```

Comment:
```suggestion
those classes. The library directory is used as an additional sourcepath for test compilation.
So jtreg implicitly compiles all test dependencies that are located in a library directory.
A test that relies upon library classes might contain appropriate
<code>@build</code> directives to explicitly compile classes.
This might be useful when a test depends on library classes during runtime only.
```